docs/content/reference/cli/wasme_build_assemblyscript.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "wasme build assemblyscript"
+weight: 5
+---
+## wasme build assemblyscript
+
+Build a wasm image from an AssemblyScript filter using NPM-in-Docker
+
+### Synopsis
+
+Build a wasm image from an AssemblyScript filter using NPM-in-Docker
+
+```
+wasme build assemblyscript SOURCE_DIRECTORY [-b <bazel target>] -t <name:tag> [flags]
+```
+
+### Options
+
+```
+  -e, --email string      Email for logging in to NPM before running npm install. Optional
```

Comment:
why are these here?

Reply:
for running `npm install` inside the container in case the user needs to.

i think a token would be better here, i can address in a follow-up